client/components/domains/transfer-domain-step/style.scss

```diff
@@ -35,34 +32,33 @@
 }
 
 .transfer-domain-step__domain-description {
+	font-size: 16px;
+	color: #2e4453;
+
 	@include breakpoint( ">960px" ) {
-		max-width: 75%;
 		float: left;
 		margin-bottom: 20px;
```

Comment:
I think this should be outside the breakpoint (which isn't really needed). Otherwise, there is no space between `transfer-domain-step__domain-description` and `transfer-domain-step__add-domain` at smaller widths. (ex: https://cloudup.com/cBxKhScyMku)